src/sinks/azure_blob/config.rs

```diff
@@ -123,11 +131,24 @@ impl AzureBlobSinkConfig {
             .blob_append_uuid
             .unwrap_or(DEFAULT_FILENAME_APPEND_UUID);
 
+        let transformer = self.encoding.transformer();
+        let (framer, serializer) = self.encoding.clone().encoding();
+        let framer = match (framer, &serializer) {
+            (Some(framer), _) => framer,
+            (None, codecs::encoding::Serializer::Json(_)) => {
+                CharacterDelimitedEncoder::new(b',').into()
+            }
```

Comment:
Is this supposed to be comma separated JSON?

Reply:
Hmm, yeah, that is surprising. I think this should be newline delimited?

Reply:
This is the mapping for `StandardEncodings`:

https://github.com/vectordotdev/vector/blob/d8b78e65d02906fedaa19b2f729b4da162c9822e/src/sinks/util/encoding/codec.rs#L184-L197

Meaning, we will only land in the `None` case if the user supplied `Json` rather than `Ndjson`.

If we fall back to newline delimited here, what's the point of the `Json` option 🤔

Reply:
Right, yeah, I see. I was neglecting that the `Ndjson` was a separate option. This makes sense to me as-is then.